examples_tests/20.Megatexture/main.cpp

```diff
@@ -64,12 +64,14 @@ struct PCstruct
     float Ni;
     uint extra; //flags copied from MTL metadata
 };
+#define nbl_glsl_MaterialParametersStruct PCstruct
```

Comment:
whats the point of this type alias anyway?

Reply:
so you don't need to override default push constants
https://github.com/Przemog1/Nabla/blob/mesh_packer/include/nbl/builtin/shader/loader/mtl/fragment_impl.glsl#L41-L43